UefiTestingPkg/FunctionalSystemTests/MemoryProtectionTest/App/MemoryProtectionTestApp.c

```diff
@@ -1833,6 +1966,88 @@ AddSmmPageTest (
   }
 } // AddSmmPageTest()
 
+STATIC
+EFI_STATUS
+DetermineTestMethod (
+  IN  MEMORY_PROTECTION_TESTING_METHOD  PreferredTestingMethod,
+  OUT MEMORY_PROTECTION_TESTING_METHOD  *TestingMethod
+  )

```

Comment:
Function description block would be nice to have.

Reply:
The `MemoryProtectionTestMax` fall through logic for determining `DeterminedTestingMethod` would particularly be helpful to summarize.

Reply:
I'll tack on an extra commit to this, convert it to a REBASE && FF, and add function headers to the whole test

Reply:
Added comments to the function in the second commit of this REBASE && FF